OpenMEEG/libs/OpenMEEG/include/gain.h

```diff
@@ -53,6 +53,7 @@ namespace OpenMEEG {
     class GainMEG : public Matrix {
     public:
         using Matrix::operator=;
+        GainMEG (const Matrix& GainMat): Matrix(GainMat) {}
         GainMEG (const SymMatrix& HeadMatInv,const Matrix& SourceMat, const Matrix& Head2MEGMat, const Matrix& Source2MEGMat) {
             *this = Source2MEGMat+(Head2MEGMat*HeadMatInv)*SourceMat;
```

Comment:
what you did if more efficient than? I think the GainEEG and GainMEG classes were introduced for the Python bindings. If you fix the constructor here it will also benefit Python users right?